src/main/java/de/maxhenkel/audioplayer/command/AudioPlayerCommands.java

```diff
@@ -379,15 +379,15 @@ private static LiteralArgumentBuilder<CommandSourceStack> applyCommand(LiteralAr
                                 })));
     }
 
-    private static LiteralArgumentBuilder<CommandSourceStack> bulkApplyCommand(LiteralArgumentBuilder<CommandSourceStack> builder, Predicate<ItemStack> validator, String itemTypeName) {
+    private static LiteralArgumentBuilder<CommandSourceStack> bulkApplyCommand(LiteralArgumentBuilder<CommandSourceStack> builder, String actionItem, Predicate<ItemStack> validator, String itemTypeName) {
```

Comment:
Passing the String `actionItem` doesn't make much sense and isn't very consistent. You could use `Predicate<ItemStack>` to validate for music discs like in the other methods.

Reply:
That pattern is so much cleaner. Latest changes uploaded